contracts/gas-snapshots/functions.gas-snapshot

```diff
@@ -52,23 +52,23 @@ FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_RevertIfNotNe
 FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_RevertIfNotOwner() (gas: 23369)
 FunctionsRouter_ProposeContractsUpdate:test_ProposeContractsUpdate_Success() (gas: 118456)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfConsumerNotAllowed() (gas: 59304)
-FunctionsRouter_SendRequest:test_SendRequest_RevertIfDuplicateRequestId() (gas: 192134)
+FunctionsRouter_SendRequest:test_SendRequest_RevertIfDuplicateRequestId() (gas: 192172)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfEmptyData() (gas: 29405)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfIncorrectDonId() (gas: 57926)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInsufficientSubscriptionBalance() (gas: 185987)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInvalidCallbackGasLimit() (gas: 50902)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfInvalidDonId() (gas: 25061)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfNoSubscription() (gas: 29111)
 FunctionsRouter_SendRequest:test_SendRequest_RevertIfPaused() (gas: 34247)
-FunctionsRouter_SendRequest:test_SendRequest_Success() (gas: 197669)
+FunctionsRouter_SendRequest:test_SendRequest_Success() (gas: 284316)
```

Comment:
This is big change! Is it just caused by the extra hash/encoding...?

Reply:
No, it is caused by the slight change to test logic itself. These gas figures are for the cost to run the Foundry test, not just the cost of our contract function calls.